protocol/avdl/keybase1/teams.avdl

```diff
@@ -39,6 +39,12 @@ protocol teams {
       KID encKID;
   }
 
+  record TeamMember {
+    UID uid;
+    string role;
```

Comment:
Can this be a proper enum type, instead of a string?  If not, where can I get the definitions of what these are allowed to be?  Are they defined as Go constants somewhere?